roles/vald/vars/main.yml

```diff
@@ -7,10 +7,6 @@ vald_environment_secrets_file_name: "{{ service_identifier }}.secrets.env"
 # binary names
 vald_default_binary_name: axelard
 
-vald_chain_id:
-  mainnet:
-  testnet:
-
-vald_validator_address:
-  mainnet:
-  testnet:
+vald_axelar_home: /opt/axelar
```

Comment:
please move this to inventory file under this host

Reply:
https://github.com/encapsulate-xyz/amplifier-ansible/pull/2#discussion_r1850048842